packages/@aws-cdk/aws-neptune-alpha/test/integ.cluster-copy-tags-to-snapshot.ts

```diff
@@ -0,0 +1,28 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as cdk from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { DatabaseCluster, InstanceType } from '../lib';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-neptune-integ');
```

Comment:
It would be good to change the stack name so that the integ test can be deployed in parallel with other tests.

Reply:
I changed the stack name.